vector-im/element-android
# [Location sharing] - Delete action on a live message (PSG-523)

## Type of change


## Content


Adding the remove action to a live location share message.

## Motivation and context


Closes #6437 

## Screenshots / GIFs





TODO

## Tests



- Ensure live location sharing is enabled in Settings -> Labs
- Go to a room
- Share a live location
- Long press on the message
- Check the "Remove" action is available
- Press the "Remove" action
- Check a dialog to confirm is displayed
- Confirm the remove
- Check the message is replaced by a "Deleted message" texts

## Tested devices

- OS version(s): Android 11

## Checklist
